### PR TITLE
Updating caption for slide 11

### PR DESCRIPTION
**Pull request by pavspatel7**
Updating Caption for Slide 11

### CODE REVIEW

File: _slides/11-pavspatel7.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
```
Comment (c4llmeco4ch) [inline _slides/11-pavspatel7.md:6]: Standard line

File: _slides/11-pavspatel7.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
+Hello, World!
```
Comment (c4llmeco4ch) [inline _slides/11-pavspatel7.md:6]: Much better!

**Review by c4llmeco4ch — APPROVED** [review]
Looks good!

File: _slides/11-pavspatel7.md (hunk 2 above)
Comment (ptalla) [inline _slides/11-pavspatel7.md:6]: reviewed to test

**Review by ptalla — APPROVED** [review]

File: _slides/11-pavspatel7.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
 
-![skitchtocat](https://octodex.github.com/images/skitchtocat.png)
+![skitchtocat](https://octodex.github.com/images/skitchtocat.png)
```
Comment (ptalla) [inline _slides/11-pavspatel7.md:8]: second review

**Review by ptalla — APPROVED** [review]

**Pull request merged**